src/main/java/frc/robot/commands/BeamBreakCommand.java

```diff
@@ -14,7 +14,9 @@
 
   public BeamBreakCommand(IntakeSubsystem intakeSubsystem) {
     // Use addRequirements() here to declare subsystem dependencies.
-    addRequirements(intakeSubsystem); 
+    if(intakeSubsystem != null) {
+      addRequirements(intakeSubsystem); 
+    }
```

Comment:
This check alone isn't helpful, because it'll just crash the first time it tries to use the intakesubsystem if it is null. 
Another PR got merged that broke the subsystem disables in RobotContainer. The shooter subsystem PR shows what should be there. 

Either remove these checks or add else { cancel() } to unschedule them if their subsystems don't exist. I'm not certain about the cancel method, so you'd have to test it if you go that route.